### PR TITLE
Automate BZ 1210541

### PR DESCRIPTION
**Pull request by sshveta**

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 22bc5aaa21257807922ea1fe1de729e8fd44c509:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 90c02eae8838896756231fa4048c44197f8f731b:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/tests/services/test_service_catalogs.py [hunk 1]
```diff
@@ -190,3 +190,16 @@ def test_no_template_catalog_item(provider_crud, provider_type, provisioning,
...
+def test_edit_catalog_after_deleting_provider(provider_crud, provider_key, provider_mgmt,
+                                              provider_init, catalog_item):
+    """Tests edit catalog item after deleting provider
+
+    Metadata:
+        test_flag: provision
+    """
+    catalog_item.create()
+    provider_crud.delete(cancel=False)
+    catalog_item.update({'description': 'my edited description'})
```
Comment (seandst) [inline cfme/tests/services/test_service_catalogs.py]: This should probably be asserting something.

**Comment by dajoRH** [comment]
Lint report for commit ae6ff720683884ff9f7600734ab5ec14ca49d18e:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 62878a8942d806a17abb130248746d06296fa5e6:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Pull request merged**